### DISCUSSION TITLE
We are looking for a Full-Stack Developer

### DISCUSSION THREAD

**Discussion by marynasteger** [community/community#134043] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi Guys!

We are WatchAdvisor AG, a profitable Swiss-based start-up renowned for producing premium digital content about luxury watches. As we evolve our business model to elevate our digital offerings, we are seeking an experienced and passionate Full-Stack Developer to join our team. If you are enthusiastic about innovative technologies and luxury timepieces, we encourage you to explore our job description!

We would appreciate every suggestion or reference to a person who would be interested in the project.

Best,

Maryna
[Developer_Stellenbeschreibung.pdf](https://github.com/user-attachments/files/16424583/Developer_Stellenbeschreibung.pdf)

**Comment by Faiaz3k** [comment]
i am interested in your job proposal kindly chake my portfolio if i am able to

**Comment by queenofcorgis** [comment]
Hi @marynasteger , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
